modules/i18n/src/translations/pt-BR/portals/common.ts

```diff
@@ -108,9 +108,9 @@ export const common: CommonNS = {
     myAccount: "Minha conta",
     name: "Nome",
     networkErrorMessage: {
-        description: "Tente recarregar o aplicativo.",
-        heading: "Quelque chose s'est mal passé",
-        primaryActionText: "Recarregue o aplicativo"
+        description: "Please try signing in again.",
```

Comment:
Is this expected?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/2771/commits/6eae9403f8dcd8d805a28eb37973a8c1df07d89c